src/IssueViz.UnitTests/Editor/Common/CommonViewTaggerProviderTestsBase.cs

```diff
@@ -128,7 +128,7 @@ public void CreateTagger_DifferentBuffers_ReturnsTaggerPerViewBuffer()
             tagger1.Should().NotBeSameAs(tagger2);
         }
 
-        private static ITextView CreateValidTextView(ITextBuffer buffer = null)
+        protected static ITextView CreateValidTextView(ITextBuffer buffer = null)
```

Comment:
If the "IsTaggable" test can be added to this class then this can remain private.